app/util/network.py

```diff
@@ -38,7 +38,8 @@ def reset_session(self):
         if self._session:
             self._session.close()  # Close any pooled connections held by the previous session.
         self._session = requests.Session()
-        self._session.mount('http://', HTTPAdapter(pool_connections=self._poolsize, pool_maxsize=self._poolsize))
+        self._session.mount('{}://'.format(Configuration['protocol_scheme']),
```

Comment:
You could make that same point about all configuration value reads. If we were to do something like that I'd rather it be consistent across the app and not just for one value.